models/workload.class.js

```diff
@@ -72,6 +79,23 @@ export default class Workload extends Resource {
     vm.$set(this, 'spec', spec);
   }
 
+  toggleRollbackModal( resources = this ) {
+    this.$dispatch('promptModal', {
+      resources,
+      component: 'RollbackWorkloadDialog'
+    });
+  }
+
+  async rollBackWorkload( workload, rollbackRequestData ) {
+    if ( Array.isArray( workload ) ) {
+      throw new TypeError(this.t('promptRollback.multipleWorkloadError'));
+    }
+    const namespace = workload.metadata.namespace;
+    const workloadName = workload.metadata.name;
+
+    await this.patch(rollbackRequestData, { url: `/apis/apps/v1/namespaces/${ namespace }/deployments/${ workloadName }` });
```

Comment:
Would be good to get the list to update after rollback - I used an example with nginx, changing the image tag - the list does not update as you point out - can you ask Vince or Richard to help - would be good to document what you had to do in order to get this working.

Reply:
Updated - Nancy explained that to make the image update in the list view, the `images` in `components/formatter/podImages.vue` needed to be moved from `data` to `computed` so that it would be updated when the row value changed. Will make doc update in separate PR